raiden/raiden_event_handler.py

```diff
@@ -216,6 +218,22 @@ def on_raiden_events(
         ]
         raiden.transport.send_async(all_messages)
 
+    @staticmethod
+    def handle_missing_metadata_events(
+        raiden: "RaidenService", event: RequestMetadata, chain_state: ChainState
+    ) -> None:
+        new_events: List[Event] = []
+        for wrapped_event in event.dependant_events:
+            metadata = fetch_address_metadata(wrapped_event.recipient)
```

Comment:
Couldn't we simply call `query_address_metadata()` directly here? I mean, we have the RaidenService object to get the pfs config and we would save us this local import on `fetch_address_metadata`

Reply:
Good catch, we don't need that anymore ✅